bouncer.go

```diff
@@ -102,6 +117,17 @@ func New(ctx context.Context, next http.Handler, config *Config, name string) (h
 	serverChecker, _ := ip.NewChecker(config.ForwardedHeadersTrustedIPs)
 	clientChecker, _ := ip.NewChecker(config.ClientTrustedIPs)
 
+	tlsConfig, err := getTLSConfigCrowdsec(*config)
+	if err != nil {
```

Comment:
I don't know if it's possible but the if need to be in validare params, no check in this function

Reply:
What if there is an error in this function ?

Reply:
handled in validate